www/package.json

```diff
@@ -57,6 +57,7 @@
     "graphql-request": "1.6.0",
     "gray-percentage": "^2.0.0",
     "hex2rgba": "^0.0.1",
+    "jest-dom": "^3.1.3",
```

Comment:
Whoops! Waiting for the tests to (hopefully) pass then will send a commit to fix this.